pkg/commonvpc/sg_analysis.go

```diff
@@ -0,0 +1,267 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package commonvpc
```

Comment:
Why is the entire file identified as new? Most of it isn't and its hard to review it like this